gui/freeadmin/static/lib/js/freeadmin.js

```diff
@@ -1416,6 +1416,22 @@ require([
 
     }
 
+    systemDatasetMigration = function() {
+        var dialog = new Dialog({
+            title: gettext("Warning!"),
+            id: "Warning_box_dialog",
+            content: domConstruct.create(
+                "p", {
+                    innerHTML: gettext(
+                        gettext("<font color='red'>Warning: This action will result in migration of dataset.<br /><br />")
+                    )
```

Comment:
gettext() should only include actual text.

Also propose to append the text: "some services will be restarted"